features/telemetry/protocols/bmp/parser/src/test/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/ParserTest.java

```diff
@@ -113,4 +114,8 @@ public void visit(RouteMonitoringPacket packet) {
     @Override
     public void visit(RouteMirroringPacket packet) {
     }
+
+    @Override
+    public void visit(final UnknownPacket packet) {
```

Comment:
All methods visit(final...?

Reply:
The whole visitor implementation is empty. I've just removed it.